.dockerignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 Dockerfile
 coverage
+**/*.test.js
```

Comment:
Using Docker ignore to exclude test files from container build to avoid need for babel transpilation.

For those devs testing in the container build the volume mount means those files will still be included in the container locally.

Reply:
@rkdefra just checking everyone is happy. You good with the above? Re: change requested. I feel as people add more items to the codebase they could also add them here `__fixtures__` for instance.

Also this isn't set in stone. I suspect as the first round of teams use this template we will get more PR's coming in. 

If happy please dismiss. Thanks